pkg/bridgetest/README.md

```diff
@@ -0,0 +1,7 @@
+# bridgetest
```

Comment:
I do. AWS used to have tests of debug output, but we have removed the ability to test this when we moved the `logging` module from `unstable` to `internal`.

This would let them add it back.

I'm happy to have that conversation in another PR. I have moved the new testing function into `unstable/testutil`.